Versions/Common/opie.lua

```diff
@@ -0,0 +1,36 @@
+local name, addon = ...
+local f = CreateFrame("Frame");
+local L = LibStub("AceLocale-3.0"):GetLocale("BeStride")
+
+-- Basedd off of Opie Rings\Default.lua file
+local function addBeStrideRingToOpie() 
+    local ringname = L["Bindings.Header"]
+    if not (OPie and OPie.CustomRings) then return end
+    local _, major = OPie.GetVersion()
+    if not (major and major == 4) then return end
+
+    OPie.CustomRings:AddDefaultRing(ringname, {
+        {"macrotext", "/click BeStride_ABRegularMount", icon="Interface/Icons/inv_misc_summerfest_brazierorange", fastClick=true},
```

Comment:
This is the Ashes of Al'ar icon but I'd be happy to change it to something else if you think it'd fit better. Maybe one of the new Dragonriding icons:

![image](https://user-images.githubusercontent.com/20718430/205437678-ec86349a-5d0f-41b8-a5c8-4d79c06281dd.png)

I also think if you update the textures here in the future (new expansions, new features) these will "just update" it a user hasn't changed them.